### DISCUSSION TITLE
GameCube adaptor wiring orientation

### DISCUSSION THREAD

**Discussion by alastairbdoggett** [darthcloud/BlueRetro#61] [Q&A] [answered]
had a question about how to interpret the GC pinout diagram relative to the GC connector port. for example, if i have the following measurements (via continuity test):

```
GC connector port
-----------------
Top left - black
Top middle - red
Top right - white
Bottom left - green
Bottom right - yellow
```

where the reference for this orientation is given by
![gc_cord_port](https://user-images.githubusercontent.com/17734670/129970538-14d8b026-3b0e-4b72-b796-7001596e9186.jpeg)

we have also have the GC pinout given by
![gamecube_pinout](https://user-images.githubusercontent.com/17734670/129970616-e1fed934-4881-4397-b623-9fdf30d9c740.png)

then, using the pinout diagram which is *facing the console* and the connector port reference photo which is *facing me*, I’m deducing the following:

```
GC connector port
-----------------
top left - black - GND
top middle - red - data (IO19)
top right - white - 5V
bottom left - green - 3V3
bottom right - yellow - GND
```

would that be a correct interpretation?

**Comment by darthcloud** [comment, marked as answer]
Yes facing the plug mirror on Y axis the facing console.

Your interpretation is correct.

I think I'll add "facing plug" diagram in the wiki in addition to the existing one.

**Comment by alastairbdoggett** [comment]
got it, thank you! oh and when i connect these to the esp32 board, do I need to group the 2x GND or am I ok just leaving one of them trailing/not connected?

**Reply by darthcloud** [reply]
Should be fine either way.

**Reply by alastairbdoggett** [reply]
perfect, thank you :)
